### PR TITLE
Update troubleshooting info for profiler 

### PR DESCRIPTION
**Pull request by juliusl**
Added troubleshooting steps for loading data in the viewer. 

@northtyphoon

### CODE REVIEW

File: app-insights-profiler-preview.md [hunk 1]
```diff
@@ -214,6 +214,19 @@ However a much simpler heuristic almost always works. Simply look at all paths.
 Since it is likely that these are simply waiting on the other threads, concentrate on the rest and ignore the time in the uninteresting threads.   
 This simple, ‘obvious’ technique, almost always works.   
 
+### <a id="issue-loading-trace-in-viewer"></a>Issue loading data in viewer
+
+There are 3 reasons we know of that could impact your ability to view your data.
+
+1. Our data retention policy has removed your trace. If the data you are trying to view is older than a couple of weeks, try limiting your time filter and try again.
+
+2. Network connectivity issues have impacted our ability process requests. Check to see if there are any azure outages that could impact our services, and wait for issues to be resolved before trying again.
```
Comment (northtyphoon) [inline app-insights-profiler-preview.md]: Check firewall/proxy if it allows connection to gateway.azureserviceprofiler.net

**Comment by SergeyKanzhelev** [comment]
Please send PR for https://docs.microsoft.com/en-us/azure/application-insights/app-insights-ip-addresses

File: app-insights-profiler-preview.md [hunk 2]
```diff
@@ -214,6 +214,19 @@ However a much simpler heuristic almost always works. Simply look at all paths.
...
 
+### <a id="issue-loading-trace-in-viewer"></a>Issue loading data in viewer
+
+There are 3 reasons we know of that could impact your ability to view your data.
+
+1. Our data retention policy has removed your trace. If the data you are trying to view is older than a couple of weeks, try limiting your time filter and try again.
+
+2. Network connectivity issues have impacted our ability process requests. Check to see if there are any azure outages that could impact our services, and wait for issues to be resolved before trying again. Also, check that proxies or a firewall has not blocked access to https://gateway.azureserviceprofiler.net. 
+
+3. There was a configuration issue when deploying the profiler. Confirm that the Application Insights instrumentation key you are using in your app (this is usually found in the web.config, app.config, etc.) is the same as the Application Insights resource you are trying to view data for.
```
Comment (SergeyKanzhelev) [inline app-insights-profiler-preview.md]: Isn't Instrumentation Key is typically in `ApplicationInsights.config`? Or in Application Settings of Azure Web App under the name `APPINSIGHTS_INSTRUMENTATIONKEY`

**Comment by SergeyKanzhelev** [comment]
@juliusl please consider adding more information about instrumentation key before I'll merge

**Comment by northtyphoon** [comment]
@SergeyKanzhelev To register the profiler endpoints in https://docs.microsoft.com/en-us/azure/application-insights/app-insights-ip-addresses, do we need to provide static ip?

**Comment by SergeyKanzhelev** [comment]
@northtyphoon not necessary, but customers will ask at some point. Here is what it used to be for live metrics: https://github.com/Microsoft/azure-docs/commit/032d32be1719d4b459c88b13616bbef8d2d7be09

File: app-insights-profiler-preview.md (hunk 2 above)
Comment (juliusl) [inline app-insights-profiler-preview.md]: Oh yeah ApplicationInsights.config that's another place. I'll add that.

**Comment by juliusl** [comment]
Added pr to azure-docs: 

https://github.com/Microsoft/azure-docs/pull/1221

**Pull request merged**